bin/flutter.bat

```diff
@@ -12,6 +12,15 @@ REM work across all platforms!
 REM
 REM --------------------------------------------------------------------------
 
+REM Execute a copy of the script to prevent modifications to a running script,
+REM see https://github.com/flutter/flutter/issues/14578 for details.
```

Comment:
I don't think you need the reference to the bug, just say something like "(as might happen during an upgrade)" or some such.